pkg/kds/global/components.go

```diff
@@ -134,7 +137,9 @@ func Setup(rt runtime.Runtime) error {
 			}
 			return
 		}
-		if err := kdsServerV2.GlobalToZoneSync(stream); err != nil {
+
+		err = kdsServerV2.GlobalToZoneSync(stream)
+		if err != nil && !errors.Is(err, context.Canceled) {
```

Comment:
some of them could be inlined - that would make me a bit less worried that we're overriding some other err

Reply:
changed